test/e2e/edgedevice_test.go

```diff
@@ -266,6 +266,11 @@ func (e *edgeDeviceDocker) Register(cmds ...string) error {
 		}
 	}
 
+	// For the docker image it's not possible to enable lingering in RPM, so we must re-do it here:
+	if _, err = e.Exec("loginctl enable-linger flotta"); err != nil {
```

Comment:
should this be part of this PR?

Reply:
I will add it to another PR, it's for modification for rootless containers to work with our e2e tests.